.pre-commit-config.yaml

```diff
@@ -31,5 +31,6 @@ repos:
     rev: v2.3.0
     hooks:
       - id: pycln
+        language_version: "3.8"
```

Comment:
Do we want to specifically say 3.8 or something like >3.8?
If we're pinning a specific version, why not the latest?

Reply:
You can't use that kind of syntax in this context.

Updated to 3.11 to match the Black hook. I won't use 3.12 because I think it's too new and there are still some projects missing new binary wheels for it.